pkg/jobs/job_info_storage.go

```diff
@@ -82,7 +82,7 @@ func (i InfoStorage) get(ctx context.Context, infoKey []byte) ([]byte, bool, err
 	row, err := i.txn.QueryRowEx(
 		ctx, "job-info-get", i.txn.KV(),
 		sessiondata.NodeUserSessionDataOverride,
-		"SELECT value FROM system.job_info WHERE job_id = $1 AND info_key = $2",
+		"SELECT value FROM system.job_info WHERE job_id = $1 AND info_key = $2 ORDER BY written LIMIT 1",
```

Comment:
Didn't you want `ORDER BY written DESC`? Presumably if there are multiple rows you want the latest one written, not the first.

Reply:
I did, fixing it up. Thanks for flagging 🙌